kube/services/jobs/psql-db-backup-encrypt-job.yaml

```diff
@@ -206,6 +211,7 @@ spec:
               # Logging the failed backups
               if [ -n "$failed_backups" ]; then
                 gen3_log_info "Failed backups:\n$failed_backups"
+                curl -X POST --data-urlencode "payload={\"text\": \"*Backup failed* for psql-db-backup-encrypt on ${hostname}  Cluster: ${environment} Namespace: ${namespace} at $(date).\nFailed backups: ${failed_backups}\"}" $slackWebHook
```

Comment:
Can we check if the slackwebhook is set to alert, and if it's not don't alert. Similar to other jobs in cloud-auto